### PR TITLE
meta-iotqa: add qa tests for librealsense

### PR DESCRIPTION
**Pull request by nvelux**
Add test for librealsense  python interface
Signed-off-by: nvelux <nagarajagopalx.velu@intel.com>

### CODE REVIEW

**Comment by nvelux** [comment]
Please help to test this.

**Comment by okartau** [comment]
ok to test

**Comment by nvelux** [comment]
corrected image path, please test it.

**Comment by mythi** [comment]
please fixup rebase the commits in one

File: meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        self.assertEqual(status, 0, msg="Error messages: %s" % output)
+        # Run an example test from librealsense-examples.
+        (status, output) = self.target.run('cpp-headless')
+        if status == 1:
+            # There might not be correct HW connected.
+            self.assertEqual(output, "There are 0 connected RealSense devices.", msg="Error messages: %s" % output)
+        else:
+            self.assertEqual(status, 0, msg="Error messages: %s" % output)
+        # Capture DEPTH image and checking image file present
+        (status, output) = self.target.run("/bin/cpp-headless")
```
Comment (mythi) [inline meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py]: Why do you need to run ```cpp-headless``` twice?

File: meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        self.assertEqual(status, 0, msg="Exe file not running: %s" % output)
+        (status, output) = self.target.run("find / -name cpp-headless-output-DEPTH.png > /tmp/log.txt")
```
Comment (mythi) [inline meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py]: the output image is written in the "current working directory". no need to search ```/ ```

Comment (nvelux) [inline meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py]: Do you know the path of output image ? 
I want to check image present or not in directory.

Comment (mythi) [inline meta-iotqa/lib/oeqa/runtime/multimedia/realsense/realsense_capture_image.py]: yes, it's "current working directory". If you ```self.target.run("cd /tmp && cpp-headless")``` the file is in ```/tmp```

**Comment by nvelux** [comment]
getting error "there are 0 connected realsense device", please can you check.

**Comment by mythi** [comment]
@nvelux that's the expected configuration

**Comment by mythi** [comment]
@nvelux please squash the commits to make the review possible

**Comment by nvelux** [comment]
squashed the commits, please review it.

**Comment by nvelux** [comment]
please review,

**Comment by mythi** [comment]
@nvelux is this test relevant as the test devices do not have cameras connected? we already have realsense_headless test that cover the first part of this module.

**Comment by nvelux** [comment]
Please merge with master branch.

**Comment by mythi** [comment]
@nvelux my question above is still not answered. In the current shape these changes have no effect and we're not planning to add the cameras. Therefore, I'll close this.

**Pull request closed without merging**